scripts/sample-data.js

```diff
@@ -111,8 +112,11 @@ async function createSampleData() {
       price: { currency: 'ETH', amount: price.toString() },
       commission: { currency: 'OGN', amount: commission.toString() },
       unitsTotal: 1
-    })
-    listings.push(await o.marketplace.getListing(newListing.listingId))
+      })
+      listings.push(await o.marketplace.getListing(newListing.listingId))
+    } catch (e){
+      
```

Comment:
Should this log an error on the console as opposed to eating silently the exception ?

Reply:
Ah, this is careless of me, was only using this for testing purposes. Anyway maybe it is a good idea to keep this in and add an error log like you propose.